code/modules/clothing/suits/labcoat.dm

```diff
@@ -30,36 +30,36 @@
 	item_state = "labcoat_green_open"
 
 /obj/item/clothing/suit/storage/labcoat/genetics
-	name = "geneticist labcoat"
+	name = "geneticist's labcoat"
```

Comment:
I dont think adding `'s` to all of these is the best idea, since theres multiple of these roles, and it leaves an inconsistency in the code.

Revert adding `'s` to the ones you did it to